tests/model/test_integrands.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+from cascade.testing_utilities import make_execution_context
+from cascade.core.context import ModelContext
+
+from cascade.model.integrands import integrand_grids_from_gbd
+
+
+def test_integrand_grids_from_gbd(mocker):
```

Comment:
a test! Yes, we have cleanup to do.

Reply:
Yeah, they are thin on the ground for this stuff. Both because I was struggling to get through the weeds and because this is the kind of code that is anti-fun to test. But we should fix that.